AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Task/UserTasks/AWSAuthUpdateUserAttributesTask.swift

```diff
@@ -36,6 +37,8 @@ class AWSAuthUpdateUserAttributesTask: AuthUpdateUserAttributesTask {
             return try await updateUserAttribute(with: accessToken)
         } catch let error as UpdateUserAttributesOutputError {
             throw error.authError
+        } catch let error as SdkError<UpdateUserAttributesOutputError> {
+            throw error.authError
         } catch let error as AuthError {
             throw error
```

Comment:
```suggestion
        } catch let error as AuthErrorConvertible {
            throw error.authError
        }
```

Can we remove the two catch and do this?

Reply:
Thanks for pointing it out.. Didn't realize that `SdkError` was conforming to `AuthErrorConvertible`. Fixed in the update.